databases/week2/diagram.sql

```diff
@@ -0,0 +1,242 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
```

Comment:
have you designed all this?

Reply:
I absolutely did, and I did not made up anything just trying to learn it as I've never been a programmer ever before, 
1 after  designing the diagram .
2. Went to forward engineer option which displays it like that, and i think that is possible on the workbench.